src/transforms/pipelines/bypass.rs

```diff
@@ -0,0 +1,75 @@
+use super::expander::ExpanderConfig;
+use crate::conditions::{not::NotConfig, AnyCondition};
+use crate::config::{DataType, ExpandType, TransformConfig, TransformContext};
+use crate::transforms::filter::FilterConfig;
+use crate::transforms::Transform;
+use indexmap::IndexMap;
+use serde::{Deserialize, Serialize};
+
+/// This transform is made to do the following trick
+///
+/// ```
+///                                 +--> filter (if condition) --> ..transforms --+
+/// event -- (expand in parallel) --+                                             +--> next pipe
+///                                 +--> filter (if !condition) ------------------+
+/// ```
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct BypassConfig {
```

Comment:
Could we add some tests for this?

Reply:
It's already tested in [this expansion](https://github.com/vectordotdev/vector/pull/9984/files#diff-967c13643e8c3f738a2676c994829132e64d81f82a88e62e98de7fc2b9792061R303) test. Do you want me to add some more? Which aspect to you want me to test in particular?

Reply:
I added a behavior test 😉

Reply:
Thanks!